### ISSUE TITLE
Class variables should have capitalized first letter

### ISSUE THREAD

**Issue by ZevEisenberg** [Raizlabs/Raizlabs-Cocoa-Style#28]
This is a convention that @mgorbach introduced me to, and I'd like to add it to the style guide:

Like this:
```objc
Class SomeClass = [someObject someMethod];
```

Not like this:

```objc
Class someClass = [someObject someMethod];
```

Note the capitalization of `SomeClass`.

**Comment by jkaufman** [comment]
Sharp. Have you seen this used anywhere else?

**Comment by mr-fixit** [comment]
What’s the argument for breaking the consistency of the ‘variable names start with lower-case’ rule?

**Comment by ZevEisenberg** [comment]
Ask @mgorbach :)

**Comment by mgorbach** [comment]
Yes, it's standard practice across most of the Apple code that I've seen. It was also at BPXL. 

Why do it? Well, class variables are fundamentally not the same kind of thing as instance variables and it makes sense to separate them semantically. You wouldn't want to confuse one for the other. This also preserves a different kind of consistency: only capitalized things should be sent class methods like `alloc`. 

Along with this, if it isn't there already, we should mention that the lack of a class is `Nil` while the lack of an object is `nil.`

@ZevEisenberg you want to PR for this? I'll merge.

**Comment by ZevEisenberg** [comment]
@mgorbach addressed both of those in #35
